sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/AddSectionsBean.java

```diff
@@ -496,4 +505,17 @@ protected void initDaysOfWeek(){
        DateFormatSymbols dfs = DateFormatSymbols.getInstance(rl.getLocale());
        daysOfWeek = dfs.getWeekdays();
     }
+
+	static void initReadOnlyCategories(ServerConfigurationService serverConfigurationService) {
+		if (readOnlyCategories == null) {
+			readOnlyCategories = new HashSet<>();
+			if (serverConfigurationService != null) {
+				String[] readOnlySectionCategories = serverConfigurationService
+						.getStrings(READ_ONLY_SECTION_CATEGORIES);
```

Comment:
Please document this new property in default.sakai.properties

Reply:
Hello, @bjones86 , I've been talking to @alejandromf about this issue as we said yesterday and it's not a new property, `section.info.readonly.section.categories`  is already defined in default.sakai.properties, you can watch it here in https://github.com/sakaiproject/sakai/blob/master/config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties#L3455

It's a permission problem, the TA rol has more permission that it should, if it's assigned to a group he/she could modify groups they no belong to (this does not happen in resources, announcements or assignments).

Thanks

Reply:
Thanks, @SedueRey